pilot/pkg/networking/util/util.go

```diff
@@ -282,6 +282,11 @@ func IsProtocolSniffingEnabledForNode(node *model.Proxy) bool {
 	return features.EnableProtocolSniffing.Get() && IsIstioVersionGE13(node)
 }
 
+func IsProtocolSniffingEnabledForInbound(node *model.Proxy) bool {
+	return IsProtocolSniffingEnabledForNode(node) ||
```

Comment:
I don't think we want this.

We want to be able to set protocol sniffing for inbound only, or protocol sniffing for everything.

The way this is set up you can do protocol sniffing for outbound only, or everything

Reply:
disregard this.. I was thinking and not or